realm/post.gno

```diff
@@ -182,27 +183,25 @@ func (post *Post) GetURL() string {
 }
 
 func (post *Post) GetGnodFormURL() string {
-	return "/r/berty/social?help&__func=AddReaction" +
-		"&userPostsAddr=" + post.userPosts.userAddr.String() +
-		"&threadid=" + post.threadID.String() +
-		"&postid=" + post.id.String() +
-		"&reaction=" + strconv.Itoa(int(Gnod))
+	return txlink.URL("AddReaction",
+		"userPostsAddr", post.userPosts.userAddr.String(),
+		"threadid", post.threadID.String(),
+		"postid", post.id.String(),
+		"reaction", strconv.Itoa(int(Gnod)))
 }
 
 func (post *Post) GetReplyFormURL() string {
-	return "/r/berty/social?help&__func=PostReply" +
-		"&userPostsAddr=" + post.userPosts.userAddr.String() +
-		"&threadid=" + post.threadID.String() +
-		"&postid=" + post.id.String() +
-		"&comment.type=textarea"
```

Comment:
Is it good to do not convert `"&comment.type=textarea`?

Reply:
I wondered the same thing. I'm following the conversion that they did for r/demo/boards. It turns out that `txlink.URL` does reflection on the function and adds the extra fields. (Pretty nice.) Here is the screenshot from 'PostReply`. Notice that the extra field is added.

![image](https://github.com/user-attachments/assets/8a5200bd-9c26-4db2-a67e-4f76e4c27233)